test/e2e/Middleware/Authorization.test.ts

```diff
@@ -0,0 +1,85 @@
+import {expect} from "chai";
+import {describe} from "mocha";
+import request from "supertest";
+import logger from "../../../src/Services/Logger";
+import {app} from "../index.test";
+import * as jwt from "jsonwebtoken";
+import config from "../../../src/Config/Config";
+
+describe("Authorization middleware tests", async () => {
+
+    it("Should return unauthorized user - user not the owner of the node", () => {
+        const token = jwt.sign({id: 777}, config.jwtKey, {expiresIn: '1h'})
+
+        try {
+            request(app.server)
+                .post("/api/user/node/diskinformation")
+                .set('Authorization', token)
+                .send({
+                    "freeSpace": "430",
+                    "takenSpace": "270",
+                    "url": "url111",
+                    "address": "address111"
+                })
+                .expect(403)
```

Comment:
For someone that doesn't have this node, this will be `expected 403 "Forbidden", got 404 "Not Found"`

so create that node first to make sure it can be fetched. 

I don't think deleting db after tests is enabled but will create an issue for that separately.